fuzz/fuzz_targets/roundtrip_miniscript_script.rs

```diff
@@ -60,4 +60,12 @@ mod tests {
         extend_vec_from_hex("007c920092935187", &mut a);
         super::do_test(&a);
     }
+
+    #[test]
+    fn test_issue_67() {
+        let mut a = Vec::new();
+        // Script corresponds to: OP_0 OP_NOTIF OP_0 OP_ELSE OP_PUSHNUM_1 OP_ENDIF
+        extend_vec_from_hex("006400675168", &mut a);
+        super::do_test(&a);
+    }
```

Comment:
Can you instead add a unit test to `src/miniscript/mod.rs`. See the `true_false` test for a template for how to round-trip test a specific script. I'd prefer not to have real unit tests in the fuzz repo.

Reply:
You are right about the test line. It just forces me to use the equivalent form `andor(0,1,0)` instead of `and_n(0,1)`.

Reply:
Weird. I tested locally and it did the opposite :)

Reply:
Travis is on my side. Haha.